lib/events.js

```diff
@@ -324,19 +328,25 @@ EventEmitter.prototype.removeListener =
         if (position < 0)
           return this;
 
-        if (position === 0)
+        if (list.length === 2)
+          events[type] = list[position ? 0 : 1];
+        else if (list.emitting) {
```

Comment:
Tiny nit: Can you add `{}` for the `if` as well if there is one for the `else` part as well?